Source/WebCore/crypto/mac/CryptoKeyECMac.cpp

```diff
@@ -23,13 +23,16 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include "config.h"
 #include "config.h"
 #include "CryptoKeyEC.h"
 
 #if ENABLE(WEB_CRYPTO)
 
 #include "CommonCryptoDERUtilities.h"
 #include "JsonWebKey.h"
+#include <pal/spi/cocoa/CoreCryptoSPI.h>
+#include <stdlib.h>
```

Comment:
Do we need stdlib.h include?

Reply:
I included it since there are types as size_t defined there.